lib/src/components/Store.test.ts

```diff
@@ -28,11 +29,34 @@ describe('Store', () => {
     expect(uut.getComponentClassForName('example.mycomponent')).toEqual(MyWrappedComponent);
   });
 
-  it('clean by component id', () => {
+  it('clean props by component id', () => {
     uut.setPropsForId('refUniqueId', { foo: 'bar' });
+    uut.cleanId('refUniqueId');
+    expect(uut.getPropsForId('refUniqueId')).toEqual({});
```

Comment:
If we cleared props - why do they equal an empty object instead of being `undefined`?

Reply:
This is an exists code and test. I don't want to change current implementation.